src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationImpl.java

```diff
@@ -49,24 +50,24 @@ public LocateMatchingApplicantsResponse locateMatchingApplicants(LocateMatchingA
         return resp;
 
     }
-
+  
     @Override
-    public ValidateApplicantServiceResponse validateApplicantService(ValidateApplicantServiceRequest lmr)
-            throws FigaroValidationServiceException {
-
-        // TODO - replace the following dummy response with a call to ORDS
-
-        // Note: Call response code and response message comes from the ORDS call.
-
-        ValidateApplicantServiceResponse resp = new ValidateApplicantServiceResponse(
-                "success",
-                0,
-                "N"
-        );
-
-        return resp;
-
-    }
+    /*
+     * service method to get the response for /validateOrgApplicantServiceOrdsResponse requests
+     */
+	public ValidateOrgApplicantServiceOrdsResponse validateOrgApplicantServiceOrdsResponse(String applPartyId,
+			String orgPartyId) throws FigaroValidationServiceException {
+
+		try {
+			return ordsapi.validateOrgApplicantService(applPartyId, orgPartyId, null);
+		} catch (ApiException ex) {
+			logger.error("Exception caught at Figaro Validation Service, validateOrgApplicantServiceOrdsResponse() : "
+					+ ex.getMessage());
+			ex.printStackTrace();
+			throw new FigaroValidationServiceException(ex.getMessage());
```

Comment:
I think we should have the cause in the custom exception or re throw the exception, by not passing the cause to the custom  exception we are loosing some context, @shaunmillargov ?

Reply:
Changed the exception message.As discussed ,passing the exception message to custom exception will be included as separate PR.

Reply:
Great. I can clean up the exception handing (add the cause into the custom exception) once the PR has been merged.